src/e2e/java/uk/gov/hmcts/reform/sscs/functional/ccd/CcdMidEventControllerFunctionalTest.java

```diff
@@ -33,14 +33,16 @@ public CcdMidEventControllerFunctionalTest() {
         useRelaxedHTTPSValidation();
     }
 
+    @DisplayName("Should return 400 for no body")
     @Test
-    public void shouldReturn400ForNoBody() throws IOException {
+    public void testNoBody() throws IOException {
```

Comment:
See comment on PreviewDocumentServiceTest re: test method naming, apply to this class also.

Reply:
ok